pkg/combustion/templates/31-elemental-register.sh.tpl

```diff
@@ -34,6 +34,9 @@ Documentation=https://github.com/rancher/system-agent
 Wants=network-online.target
 After=network-online.target
 After=time-sync.target
+Wants=dbus.service
+After=dbus.service
+BindsTo=dbus.service
```

Comment:
I'm really conscious of establishing such a strong dependency on dbus. Can we get away with `Requires=` instead and / or depending on the dbus.socket rather than the service?